venues/tests/test_api.py

```diff
@@ -56,6 +59,9 @@ def autouse_db(db):
 VENUE_QUERY = """
 query Venue($id: ID!) {
   venue(id: $id){
+    name
+    description
+    address
```

Comment:
Is there a reason why these are not using all the fields?

Reply:
No specific reason at all, just too lazy to add because the snapshot might be too long